### PR TITLE
Csv fix

### PR DESCRIPTION
**Pull request by nitisht**

### CODE REVIEW

**Review by harshavardhana — DISMISSED** [review]

File: manifests/minio-operator.v4.1.2.clusterserviceversion.yaml [hunk 1]
```diff
@@ -8,37 +8,66 @@ metadata:
...
+                "name": "minio-creds-secret"
+              },
+              "pools": [
+                {
+                  "servers": 4,
+                  "name": "pool-0",
+                  "volumesPerServer": 4,
+                  "volumeClaimTemplate": {
+                    "metadata": {
+                      "name": "data"
```
Comment (ravindk89) [inline manifests/minio-operator.v4.1.2.clusterserviceversion.yaml:34]: Minor not, but if this has no other effect beyond visibility, can we prefix with "minio-" so the resulting PVC are more easily associated with MinIO?

Disregard if this field needs to be data.

**Review by ravindk89 — DISMISSED** [review]
Two nits but not blocking. LGTM whether or not they are addressed.

File: manifests/minio-operator.v4.1.2.clusterserviceversion.yaml [hunk 2]
```diff
@@ -8,37 +8,66 @@ metadata:
...
+                    "runAsNonRoot": true,
+                    "fsGroup": 1000
                   }
                 }
-              },
-              "credsSecret": {
-                "name": "minio-creds-secret"
-              },
+              ],
               "requestAutoCert": false,
```
[the remark this answers is not in the document]
Comment (nitisht) [reply manifests/minio-operator.v4.1.2.clusterserviceversion.yaml]: Makes sense to make it true by default as we do in other examples

File: manifests/minio-operator.v4.1.2.clusterserviceversion.yaml (hunk 1 above)
Comment (nitisht) [reply manifests/minio-operator.v4.1.2.clusterserviceversion.yaml]: makes sense, renamed to `minio-`.

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**